guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,389 @@
+
+Static addresses & labels enable robust [contacts](/guide/how-it-works/silent-payments/#contacts) features. The result is a better interaction model centred around users.
+
+
+### Rationale
+
+Since the bitcoin blockchain is public, reusing an on-chain address informs the network that these payments are made to the same user. However, specifying a new address for each transaction usually requires user interaction. This is error-prone, takes time and requires manual effort.
+
+Silent payments circumvent both these issues by using static addresses (starting with `sp1`).
+
+## How silent payments work
+
+Silent payments are made in 4 broad steps:
+1. The receiver shares/publishes a static payment address (manual, one-time)
+2. The sender obtains & uses it to derive a unique on-chain address (partly automated)
+3. The sender broadcasts a transaction that pays this derived address (manual)
+4. The receiver scans the blockchain & identifies their payments (automated)
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work-mobile@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "Alice posts a static address on the website of her non-profit, Chasing Rabbits. Bob uses it to send bitcoin from his wallet. This transaction pays a derived on-chain address that Alice can recognise while scanning the blockchain."
+   width = 1024
+   height = 640
+   layout = "full-width"
+%}
+
+## A new conceptual model
+
+Silent payments introduce a new technical model that separates payment information based on their purpose.
+
+For example: the static address is only shared with senders who use it to make payments. Payment detection is done with a separate scan key and doesn't use the static address at all. Here are the main components of this new model:
+
+| Component       |Used by         | Purpose |
+|:-------------|:------------------|:------|
+| Static address | Senders | Making payments
+| Scan key     | Nodes & [light clients](/guide/how-it-works/nodes/#light-nodes)  | Detecting payments
+| Spend key   | Wallet owner | Spending bitcoin
+
+
+The above model introduces a scan key that is used for payment scanning and a static address which is used for deriving on-chain addresses. This is in contrast with the BIP-32 model where [extended public keys](/guide/glossary/#extended-public-key-xpub-ypub-zpub) are used for both purposes. Below is a summary of the differences between the two models:
+
+| BIP-32       | Silent payments          | Benefit  |
```

Comment:
Still don't think this table works very well. You get one word per line on mobile, and desktop is also not that nice. No way to format this as text instead?

<img width="846" alt="image" src="https://github.com/user-attachments/assets/53e72dc8-0762-46c9-8508-a0b51326a138">

![IMG_8862](https://github.com/user-attachments/assets/f13a4a4e-db8a-4e60-a1aa-6cc02a405f14)